0_portfolio_input_check_functions.R

```diff
@@ -1092,6 +1092,142 @@ process_raw_portfolio <- function(portfolio_raw,
   return(portfolio_total)
 }
 
+get_fund_coverage <- function(portfolio_raw,
+                              fin_data,
+                              fund_data,
+                              currencies,
+                              grouping_variables) {
+  portfolio <- clean_colnames_portfolio_input_file(portfolio_raw)
+  
+  portfolio <- clear_portfolio_input_blanks(portfolio)
+  
+  portfolio <- add_meta_portfolio(portfolio, inc_meta_portfolio)
+  
+  start_port_rows <- nrow(portfolio)
+  
+  portfolio <- add_holding_id(portfolio)
+  
+  portfolio <- check_missing_cols(portfolio, grouping_variables)
+  
+  portfolio <- clean_portfolio_col_types(portfolio, grouping_variables)
+  
+  portfolio <- convert_currencies(portfolio, currencies)
+  
+  cols_portfolio <- colnames(portfolio)
+  
+  cols_of_funds <- c("direct_holding", "fund_isin", "original_value_usd")
+  
+  # Add financial data
+  # Merges in the clean data and calculates the marketvalue and number of shares
+  portfolio <- add_fin_data(portfolio, fin_data)
+  
+  if (nrow(portfolio) != start_port_rows) {
+    stop("Portfolio lines changing unexpectedly")
+  }
+  
+  portfolio <- calculate_value_usd_with_fin_data(portfolio)
+  
+  portfolio <- calculate_number_of_shares(portfolio)
+  
+  original_value_usd <- sum(portfolio$value_usd, na.rm = T)
```

Comment:
it's best practice to use `TRUE` instead of `T`

Reply:
changed in the new commit

Reply:
changed here. Flag: this is also "copied" code. T is used throughout the code

Reply:
this has been fixed